components/server/src/auth/verification-service.ts

```diff
@@ -58,6 +59,24 @@ export class VerificationService {
         return user;
     }
 
+    public async verifyOrgMembers(organizationId: string): Promise<void> {
+        const members = await this.teamDB.findMembersByTeam(organizationId);
+        for (const member of members) {
+            const user = await this.userDB.findUserById(member.userId);
+            if (user) {
+                await this.verifyUser(user);
+            }
+        }
+    }
```

Comment:
What if we used the `TeamService` here instead to list the members? The benefit I can see is we're not duplicating the logic on fetching the users and (should we need to adjust the returned data, we do that in a single place). It would also remove the dependency of the Verification Service on the Team DB directly

Reply:
you mean the grpc api?

Reply:
No, not grpc. I just checked and there isn't one for Teams/Orgs, but something like the `UserService`. The rough outline is that it would have 

```
interface OrganizationsService {
  listMembers()
  get(id)
  delete(id)
}
```

The main benefit I see is when go and move these into gRPC, we'll need to share the implementation between the websocket and gRPC at least for the duration of the migration but it would also give us a place to co-locate operations done on Organizations, which are not strictly just DB operations

Reply:
I believe TeamDB provides that level of abstraction already

Reply:
I'll do an example in a follow up PR to show the duplication, and how it would help in particular with keeping the implementation between the grpc and websocket APIs the same